src/Sulu/Bundle/AutomationBundle/Resources/config/services.xml

```diff
@@ -9,5 +9,23 @@
             <tag name="sulu.admin"/>
             <tag name="sulu.context" context="admin"/>
         </service>
+
+        <service id="sulu_automation.tasks.manager" class="Sulu\Bundle\AutomationBundle\Tasks\Manager\TaskManager">
+            <argument type="service" id="sulu.repository.task"/>
+            <argument type="service" id="event_dispatcher"/>
+        </service>
+
+        <!-- FIXME hack to enable doctrine object constructor -->
+        <!--
+          could be fixed in serializer by changing JMS\SerializerBundle\DependencyInjection\Compiler\DoctrinePass:49
```

Comment:
we would have to update jms-serializer first. before that it makes less sense to fix it there.